darwin/dataset/upload_manager.py

```diff
@@ -351,7 +339,7 @@ def upload_annotations(
                             client=client,
                             team=team,
                             annotation_type_ids=[
-                                "1"
+                                "3"
```

Comment:
no magic numbers :)

Reply:
A way of avoiding magic numbers is fetching annotation types from the backend, but since they're _rather_ static, we could just replicate an annotation type id/name mapping and use it.
```python
annotation_types = {..., 'polygon': 3, ...}
```